.rubocop.yml

```diff
@@ -26,23 +23,8 @@ Metrics/MethodLength:
   Exclude:
     - "spec/features/*"
 
-Style/ClassAndModuleChildren:
-  Enabled: false
-
-Style/Documentation:
-  Enabled: false
-
-Style/DoubleNegation:
-  Enabled: false
-
 Style/StringLiterals:
   Enabled: false
```

Comment:
This PR introduces no new rules, it just converts the old file to use the shared base. That rule was introduced by @scmx if I remember correctly.